pandas/core/indexes/multi.py

```diff
@@ -2628,13 +2628,25 @@ def _partial_tup_index(self, tup, side="left"):
                     loc -= 1
                 return start + section.searchsorted(loc, side=side)
 
-            idx = lev.get_loc(lab)
+            idx = self._get_loc_single_index_wrapper(lev, lab)
             if k < n - 1:
                 end = start + section.searchsorted(idx, side="right")
                 start = start + section.searchsorted(idx, side="left")
             else:
                 return start + section.searchsorted(idx, side=side)
 
+    def _get_loc_single_index_wrapper(self, level_index, key):
```

Comment:
name these parameters how they are called (level, label), pls add type annotations and a full-docstring.